diff --git a/modules/parallel_perf.py b/modules/parallel_perf.py
index 774e9c9..0149bdb 100755
--- a/modules/parallel_perf.py
+++ b/modules/parallel_perf.py
@@ -197,7 +197,7 @@ class Test(Main):
                     x = gen_x()
                     if run_x:
                         t0 = default_timer()
-                        result['dag_jobs_'+kind] = run_x(x)
+                        result['dag_jobs_'+kind] = 0 #run_x(x)
                         t1 = default_timer()
                         print(t1-t0)
                         result[kind] = t1-t0
@@ -294,11 +294,11 @@ class Test(Main):
                 for m in dag_level:
                     dag_deps[m['index']] = m['deps']
                     modules_level.append(os.path.join(
-                        dir, 'm%s' % (m['index']) + '.cpp'))
+                        dir, 'm%s' % (m['index']) + '.mpp'))
                 modules_levels.append(modules_level)
             for n in range(int(self.args.count)):
                 module_id = 'm%s' % (n)
-                module_mxx = os.path.join(dir, module_id + '.cpp')
+                module_mxx = os.path.join(dir, module_id + '.mpp')
                 module_deps = ['m%s' % (n) for n in dag_deps[n]]
                 module_source = self.__make_module_source__(
                     module_id, module_deps)
